### DISCUSSION TITLE
are you planning to add support for the Preferences library to the ArduinoEEPROMAbstraction on esp32?

### DISCUSSION THREAD

**Discussion by emaayan** [davetcc/tcMenu#359] [Q&A]
from what i understand EEPROM library is considred deprecated so perhaps it would make sense to add support for that/

**Comment by davetcc** [comment]
Please can you create an issue in https://github.com/davetcc/tcMenuLib/issues with a few example sketches that I can use to get an idea of how it works? Hopefully should be straightforward enough.

**Reply by vzahradnik** [reply]
It looks straightforward, indeed.
https://randomnerdtutorials.com/esp32-save-data-permanently-preferences/

**Comment by vzahradnik** [comment]
Created an issue here:
https://github.com/davetcc/tcMenu/issues/378
